site/mailing-lists.html

```diff
@@ -12,7 +12,7 @@
 
   
     <meta http-equiv="refresh" content="0; url=/community.html">
-    <link rel="canonical" href="https://spark.apache.org/community.html" />
+    <link rel="canonical" href="http://localhost:4000/community.html" />
```

Comment:
this shouldn't change?

Reply:
Yea, IIRC, this happens when jeykill is ran with serve.